wazo_call_logd/tests/test_generator_scenarios.py

```diff
@@ -772,3 +772,224 @@ def test_bad_apple(self, cels: list[CEL]):
+            CHAN_END                  | 1738431122.458 | 1738431114.455 | 2025-02-01 21:02:10.601069+03:30 |                            | 8083339999                           | s                                    | agentcallback                                             | Local/id-1@agentcallback-00000088;1                                   | AppQueue        | (Outgoing Line)                                                                                                                              |
+            BRIDGE_EXIT               | 1738431122.459 | 1738431114.455 | 2025-02-01 21:02:10.601229+03:30 | John Conner                | 0012345678910                        | id-1                                 | agentcallback                                             | Local/id-1@agentcallback-00000088;2                                   | Dial            | Local/b464fe7b-bf50-417d-a14b-1bd276f83bf1@usersharedlines,,                                                                                 | {"bridge_id":"dd551bc7-af1b-433b-aa91-bc5cd6b01200","bridge_technology":"simple_bridge"}
+            HANGUP                    | 1738431122.459 | 1738431114.455 | 2025-02-01 21:02:10.601576+03:30 | John Conner                | 0012345678910                        | id-1                                 | agentcallback                                             | Local/id-1@agentcallback-00000088;2                                   |                 |                                                                                                                                              | {"hangupcause":16,"hangupsource":"","dialstatus":"ANSWER"}
+            CHAN_END                  | 1738431122.459 | 1738431114.455 | 2025-02-01 21:02:10.601576+03:30 | John Conner                | 0012345678910                        | id-1                                 | agentcallback                                             | Local/id-1@agentcallback-00000088;2                                   |                 |                                                                                                                                              |
+            BRIDGE_EXIT               | 1738431123.475 | 1738431114.455 | 2025-02-01 21:02:23.441628+03:30 | John Conner                | 0012345678910                        | s                                    | ctx-Company-internal-3344220c-6383-4581-9696-15055ec303e3 | PJSIP/p4go4oxa-00000036                                               | Stasis          | dial_mobile,join,2d3a3a36-93d1-4a6c-8a58-aec01aea8468                                                                                        | {"bridge_id":"wazo-dial-mobile-2d3a3a36-93d1-4a6c-8a58-aec01aea8468","bridge_technology":"simple_bridge"}
+        '''
```

Comment:
This scenario is more complex than necessary. Notably, there are mobile dispatches that add irrelevant noise. The IVR is also not relevant.

The CEL entries also do not seem correctly ordered.

A minimal representative scenario is expected:
- an incall
- directed to a queue destination
- that is answered by a queue member(user or agent, doesn't really matter)

The queue member should have a single SIP line and no mobile sessions.

After reproducing such a scenario, the CEL table can be queried, e.g.:
```
select eventtype, eventtime, cid_name, cid_num, cid_ani, exten, context, channame, linkedid, uniqueid, extra from cel where call_log_id = %call_log_id order by eventtime asc;
```
(note the column list is restricted to relevant columns, and the `order by` clause)